run_container.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Public Functions
+
+// RunContainer runs a container with given ID
+func RunContainer(containerID string, containerPath string) error {
```

Comment:
I knew system containers were systemd-related so I was assuming we were using nspawn - oops. These are definitely system containers, then.